src/ctia/schemas/core.clj

```diff
@@ -1,23 +1,31 @@
 (ns ctia.schemas.core
-  (:require [ctim.schemas
-             [verdict :as vs]
-             [bundle :as bundle]
-             [common :as cos]
-             [vocabularies :as vocs]]
-            [flanders
-             [schema :as f-schema]
-             [spec :as f-spec]
-             [utils :as fu]]
-            [schema-tools.core :as st]
-            [schema.core :as sc :refer [Bool Str]]))
-
-(sc/defschema ACLEntity
-  (st/optional-keys
-   {:authorized_users [Str]
-    :authorized_groups [Str]}))
-
-(sc/defschema ACLStoredEntity
-  (st/merge ACLEntity
+  (:require
+   [ctim.schemas
+    [verdict :as vs]
+    [bundle :as bundle]
+    [common :as cos]
+    [vocabularies :as vocs]]
+   [flanders
+    [core :as flanders]
+    [schema :as f-schema]
+    [spec :as f-spec]
+    [utils :as fu]]
+   [schema-tools.core :as st]
+   [schema.core :as sc :refer [Bool Str]]
+   [schema.core :as s]))
+
+(sc/defschema OpenCTIMSchemaVersion
+  {(sc/optional-key :schema_version) s/Str})
+
+(sc/defschema CTIAEntity
+  (st/merge
+   OpenCTIMSchemaVersion
+   (st/optional-keys
+    {:authorized_users [Str]
+     :authorized_groups [Str]})))
```

Comment:
Why use two `defschema` declaration here? Knowing that `OpenCTIMSchemaVersion` is only used here? Why not just

~~~clojure
(sc/defschema CTIAEntity
  (st/optional-keys
    {:authorized_users [Str]
     :authorized_groups [Str]
     :schema_version s/Str}))
~~~

Reply:
I wanted to make it clear and obvious if we decide otherwise later.